traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/dns/protocol/TCP.java

```
@@ -99,6 +100,8 @@ public void run() {
                 final byte[] response = query(client, request);
                 os.writeShort(response.length);
                 os.write(response);
+            } catch (final WireParseException e) {
+                LOGGER.info(e.getMessage());
```

I don't follow. The exception is logged at INFO which means it will be in both the access log and application log anyway; if you wanted to avoid duplication, wouldn't you want to swallow the exception or log it at a level other than INFO? In any case, INFO is not correct for an error?